### DISCUSSION TITLE
How to watch store value in another store

### DISCUSSION THREAD

**Discussion by Sulaymon-dev** [alpinejs/alpine#2249] [1. Help] [answered]
I have two store 'breakpoints' and 'global';
In breakpoints.js I stored "currentBreakpoint, height, width" and ... ;
I what to watch breakpoints.js properties in global.js;
How can I do that?

**Comment by Sulaymon-dev** [comment, marked as answer]
use Alpine.effect inside on init method
```JavaScript
 init() {
        Alpine.effect(() => {
            console.log(Alpine.store('breakpoints').width)
        })
    },
```
Simple ;-)

**Reply by rexwebmedia** [reply]
this should be in the documentation, maybe in the Store section.
